test/helpers/utils.ts

```diff
@@ -446,3 +446,38 @@ export function encodePath(tokenAddresses, fees) {
 
   return encoded.toLowerCase();
 }
+
+/* eslint @typescript-eslint/no-explicit-any: "off" */
+export const objectEquals = (a: any, b: any) => {
```

Comment:
nit: I'd prefer if we used lodash for common utility functions rather than copying functions into the `utils.ts`

https://lodash.com/docs/4.17.15#isEqual